consoles/sshVirtsh.pm

```diff
@@ -136,8 +136,14 @@ sub _init_xml {
     }
 
     if (get_var('UEFI') and check_var('ARCH', 'x86_64') and !get_var('BIOS')) {
-        # These are known locations for openSUSE and Fedora (respectively).
-        my @known = ('/usr/share/qemu/ovmf-x86_64-ms.bin', '/usr/share/edk2.git/ovmf-x64/OVMF_CODE-pure-efi.fd');
+        # These are known locations: first is openSUSE, second is for
+        # the kraxel.org nightly packages, third is Fedora's edk2-ovmf
+        # package.
+        my @known = (
+            '/usr/share/qemu/ovmf-x86_64-ms.bin',
+            '/usr/share/edk2.git/ovmf-x64/OVMF_CODE-pure-efi.fd',
+            '/usr/share/edk2/ovmf/OVMF_CODE.fd',
```

Comment:
that's something I planned for a long time (and I guess also others did) but did never manage to complete so far, see https://github.com/os-autoinst/os-autoinst/pull/628 for my start to extract stuff from `bmwqemu` with a sensible name. Feel free to continue there and do it properly as your perl skills probably exceed mine :-)